Global/JetBrains.gitignore

```diff
@@ -44,6 +44,16 @@ cmake-build-*/
 # IntelliJ
 out/
 
+# Eclipse
+.classpath
+.project
+.settings/
+
+# Intellij
+.idea/
+*.iml
```

Comment:
We have a section earlier in the template that talks about additional rules that can be enabled when you are using Maven, which might be relevant:

https://github.com/github/gitignore/blob/7eef17f37c63ce3cbddbdd154ff836f370d0ad70/Global/JetBrains.gitignore#L27-L33